Kernel/style.m

```diff
@@ -98,9 +105,18 @@
     "ImageSizePerPlotRange" -> $ruleImageSizePerPlotRange
   |>,
 
-  (* For future use *)
+  (* MultiwaySystem styles *)
+  
+  "StatesGraph" -> <|
+    "VertexStyle" -> $statesGraphVertexStyle,
+    "EdgeStyle" -> $statesGraphEdgeStyle
+  |>,
+  "EvolutionCausalGraph" -> <|
```

Comment:
Don't we need a vertex style(s) here? And for the branchial graph?

Reply:
I don't think so, since the two relevant vertex styles (for state and event vertices) can be inherited from the state and causal graph cases. Likewise for the branchial graph. Do you think it's worth repeating them here?

Reply:
I think it is, for the sake of clarity. We can point them to the same variables (i.e., `"StateVertexStyle" -> $statesGraphVertexStyle`, etc.). But keep in mind, `WolframPhysicsProjectStyleData` is exposed to users, and I think it will be hard for people to figure out where did the vertex style come from if it's not there.

Reply:
Isn't it better to call them `"EventVertexStyle"` and `"StateVertexStyle"`? One could argue events are also part of the evolution...